fix: kill postgrest main thread on admin server crash (#5109)
PostgREST/postgrest: CHANGELOG.md

- Stop reporting 503s errors unnecessarily while the schema cache is loading at startup by @mkleczek in #4880
- Fix responding with `Something went wrong` on Admin server when under EMFILE by @mkleczek in #5077
- Fix schema cache dump missing RPC transaction isolation level by @taimoorzaeem in #5079

> - Fix admin server crash not killing PostgREST main thread by @taimoorzaeem in #5096

Reviewer: ```suggestion
- Fix admin server crashing without a way to recover by @taimoorzaeem in #5096 
```
